mobile/README.md

```diff
@@ -64,7 +63,15 @@ The example application accomplishes the task of recovering a high resolution (H
 - [iOS Super Resolution](examples/super_resolution/ios)
 
 ### Question Answering
+
 The example app gives a demo of introducing question answering models with pre/post processing into mobile scenario. Currently supports on platform Android and iOS.
 
 - [Android Question answering](examples/question_answering/android)
 - [iOS Question answering](examples/question_answering/ios)
+
+### Whisper (Cloud/Local)
+
+This example shows how to use ORT to do speech recognition using the [Whisper](https://github.com/openai/whisper) model. One version (Cloud) calls the OpenAI Whisper endpoint using the Azure custom op. (MUST update the code to specify a valid OpenAI auth token. ). The other uses a local Whisper model.
```

Comment:
nit: I wouldn't include the info about the auth token here as this file seems to mainly provide high level descriptions of the examples. The need to create the token is covered by the readme for the actual example.

Reply:
removed.